sematic/examples/template/__main__.yaml

```diff
@@ -0,0 +1,10 @@
+version: 1
+base_uri: # The URI for your base image
```

Comment:
Maybe we should automatically include `sematicai/sematic-worker-base`, in order to simplify the build config.

Reply:
good idea